includes/admin/class-wc-admin-assets.php

```diff
@@ -156,6 +156,7 @@ public function admin_scripts() {
 				'i18_sale_less_than_regular_error'  => __( 'Please enter in a value less than the regular price.', 'woocommerce' ),
 				'decimal_point'                     => $decimal,
 				'mon_decimal_point'                 => wc_get_price_decimal_separator(),
+				'hexa_color_error'					=> __( 'Invalid hexa color', 'woocommerce' ),
```

Comment:
Invalide hex color :)

Reply:
@shivapoudel ,

Thanks for your suggestions, as per your suggestions I have updated the changes please check for the same and let me know if any changes required.

Thanks.